(DOCSP-26322) Added an IP access list restriction (#98)
mongodb/docs-atlas-cli: source/connect-atlas-cli.txt

      - :ref:`Install the {+atlas-cli+} <install-atlas-cli>`.
      - Add your host's IP address to the :ref:`IP access list <access-list>`.
  
        If you authenticate with your |service| user credentials
        and your organization's owners enable :atlas:`IP access list for the Atlas UI for an organization
        </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`,
        your IP address must be added to the IP access list to run commands
        in this organization. To learn more, see :atlas:`Require IP Access List for the Atlas UI

>         </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`.

Reviewer: Not sure if this is a config issue, but when clicked this link it actually took me to [here](https://www.mongodb.com/docs/atlas/tutorial/manage-organizations/?_ga=2.25262141.1031606856.1676317638-1380600391.1667244526#require-ip-access-list-for-the-atlas-ui), but the page didn't have the instructions on how to set up ip access list. I remembered UI IP access list related doc only lives in staging and not publicly accessible?

Author: Yes. This link won't work until both PRs are published. I will not merge or publish either IP access list PRs until the feature goes GA. So this link will work once things are published.

Author: Please approve. I will keep the PR in the Ready For Merge state until the feature goes live.

      - :ref:`Install the {+atlas-cli+} <install-atlas-cli>`.
      - Add your host's IP address to the :ref:`IP access list <access-list>`.
  

>         If you log into the {+atlas-cli+} with your Atlas user credentials,

Reviewer 2: Nit: they don't log in to the CLI, they log in to Atlas using the CLI. Also, no comma needed here
```suggestion
        If you authenticate with your |service| user credentials
```

        and your organization's owners enable
        :atlas:`IP access list for the Atlas UI for an organization
        </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`,

>         then to run any commands in this organization, your IP address must
>         be included in the IP access list or CIDR ranges defined in that list.

Reviewer 2: Suggestion: simplifies the text here ("must be added" could be either as a single IP address or CIDR range)
```suggestion
        your IP address must be added to the IP access list to run 
        commands in this organization.
```
